eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/boot/EventMeshAdminServer.java

```diff
@@ -0,0 +1,135 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.eventmesh.runtime.boot;
+
+import org.apache.eventmesh.common.utils.LogUtils;
+import org.apache.eventmesh.runtime.admin.controller.HttpHandlerManager;
+import org.apache.eventmesh.runtime.configuration.EventMeshHTTPConfiguration;
+
+import io.netty.bootstrap.ServerBootstrap;
+import io.netty.channel.ChannelHandlerContext;
+import io.netty.channel.ChannelInboundHandlerAdapter;
+import io.netty.channel.ChannelInitializer;
+import io.netty.channel.ChannelOption;
+import io.netty.channel.ChannelPipeline;
+import io.netty.channel.epoll.EpollServerSocketChannel;
+import io.netty.channel.socket.SocketChannel;
+import io.netty.channel.socket.nio.NioServerSocketChannel;
+import io.netty.handler.codec.http.HttpObjectAggregator;
+import io.netty.handler.codec.http.HttpRequest;
+import io.netty.handler.codec.http.HttpRequestDecoder;
+import io.netty.handler.codec.http.HttpResponseEncoder;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public class EventMeshAdminServer extends AbstractHTTPServer {
+
+    private HttpConnectionHandler httpConnectionHandler;
+
+    HttpHandlerManager httpHandlerManager;
+
+    public EventMeshAdminServer(int port, boolean useTLS,
+        EventMeshHTTPConfiguration eventMeshHttpConfiguration, HttpHandlerManager httpHandlerManager) {
+        super(port, useTLS, eventMeshHttpConfiguration);
+        this.httpHandlerManager = httpHandlerManager;
+    }
+
+    @Override
+    public void init() throws Exception {
+        String threadPrefix = "eventMesh-admin-http";
+        buildBossGroup(threadPrefix);
+        buildIOGroup(threadPrefix);
+        buildWorkerGroup(threadPrefix);
+        httpConnectionHandler = new HttpConnectionHandler();
+    }
```

Comment:
`EventMeshAdminServer` inherits from `AbstractHTTPServer`, so there is no need to skip a level of inheritance and use the constructor of `AbstractRemotingServer`. You can refer to the `init` method of `EventMeshHTTPServer` and create `EventMeshAdminServer` as a lightweight replica of `EventMeshHTTPServer`.